### ISSUE TITLE
New metric: MCC

### ISSUE THREAD

**Issue by lazappi** [theislab/atlas-feature-selection-benchmark#14]
Thanks for taking the time to suggest a new metric!

## Description

> _Please briefly describe the suggested metric: What is it? What does it measure? Why would it be a good fit for the project?_

The Matthews Correlation Coefficient (MCC) measures the quality of binary and multiclass classifications and is regarded as a balanced measure that can be used even if the classes are of very different sizes.

## Links

- **Description:** https://scikit-learn.org/stable/modules/generated/sklearn.metrics.matthews_corrcoef.html
- **Code:** https://github.com/scikit-learn/scikit-learn
- **Package:** https://pypi.org/project/sklearn/

## Anything else?

> _Anything else you think is important about the metric_
